sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -108,6 +108,15 @@ public Schemas(Settings settings,
         return (T) info;
     }
 
+    public void validateTableName(TableIdent ident) {
```

Comment:
getTableInfo() calls blockedRaiseException which will throw an exception that is not handled by the ExceptionAuthorizedValidator and that the user might not be allowed to see